### PR TITLE
Add “Trust levels” document

### PR DESCRIPTION
**Pull request by wchargin**
Summary:
Imported from Discourse, and changed to remove requests for comment:
<https://discourse.sourcecred.io/t/trust-levels/490>

Paired with @decentralion.

wchargin-branch: trust-levels

### CODE REVIEW

**Comment by mzargham** [comment]
**Review Conclusion**
Comfortable publishing as is, but would support a minor update to ensure clearer disambiguation from discourse guidelines, and I'd like to see us acknowledge explicitly the difference between exploitive behavior and divergence in the interpretation of community values/norms.

**Detailed Comments**
Item 1: Comparison to the Usage of the Term 'Trust Levels' in Discourse.
I think it would be prudent distinguish between individual trust levels as defined in discourse, from community trust levels. The latter is an emergent property, and I suspect that there is a very real case where every individual is has relatively high trust and yet the community is subject to adversarial behavior. I would suggest we append a term to distinguish, "Ambient Trust Level" or something of the like. I do not wish to imply that any specific actor is low trust but as a community scales the degree to which normative social feedback mechanisms function to limit exploitation loses power.

Item 2: Specific definitions of the levels 3,2,1
I concur with this breakdown of level 3,2,1. Only improvement to the document would be long form descriptions of examples of behavior exemplifying the stages. However, i think it is right to no include thes in the core document but rather as supplement material (which can be added later). The main reason for long form examples is to ensure understanding of these key concepts by persons who join later and have not been privy to discussions that fleshed out the concepts, which are now distilled here.

Item 3: SourceCred the framework versus SourceCred the instance
Its wholly evident to me that sourcecred the framework must support all 3 levels discussed and it is very likely that some project may onboard sourcecred tools while already in level 2 or 1. SourceCred's instance of sourcecred is dogfooding in level 3, we may need to temper our conclusions and recommendations to others with the recognition that some insights may not extrapolate well to instances in lower ambient trust levels. For the purpose of the document, i think the writing is clear, but I would not object to a sentence expounding on this point.

Item 4: Disputes & judgements
Document contains not consideration for disputes. It is natural especially as the community grows for the value systems of individuals to be less aligned. It is my hope that the general direction of the communities value system will serve as a beacon, but it is natural that individuals will interpret the same value systems different enough to make divergent judgements on the same behavior. While it is not necessary to explicitly discuss disputes and how judgement should be passed on them. I believe it is prudent to note that as the community grows even if the (mathematical) variance around what is considered appropriate behavior holds constant, the observed behaviors will grow wider. One aspect of the trust levels concept is that such observation does not necessarily mean that attackers are malicious. One way of framing this is that in level 2, the observed variance of perceived appropriate behavior is sufficient to provide conflict, that property is only more likely in level 1, but 1 also introduces the risk of explicit exploitation by those who might feign adherence for the purpose of extracted resources whose value is recognized outside the immediate community.

File: culture/trust_levels.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+This post proposes a system of **trust levels** for SourceCred the
+system, which describe how we intend to handle the very real questions
+of adversarial actors and gaming resistance over different time scales.
+
```
Comment (vsoch) [inline culture/trust_levels.md]: Has it been referred to as "SourceCred the system?" What exactly is a system here - an implementation, or an interface, or something else? I think it would read cleanly to remove the system (and people know we are talking about SourceCred) or to be more specific about what "the system" is. Am I entering the matrix? My web browser?

File: culture/trust_levels.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+The ideas proposed herein have been discussed informally within our
+community, but with somewhat confusingly varying semantics. It may help
+to have this shared reference to point to.
+
```
Comment (vsoch) [inline culture/trust_levels.md]: Can this be said more simply? I would suggest:

> The ideas proposed here have been discussed informally, and this shared document is provided for summary and reference.

File: culture/trust_levels.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+**Note:** The term *trust levels* as used in this post is largely
+separate from [Discourse trust levels][discourse-trust], which are a
+construct of our forum software and not related to the SourceCred
+protocol in any way.
```
Comment (vsoch) [inline culture/trust_levels.md]: Oh that's really interesting - I'm familiar with Discourse trust level and just assumed they were similar. Could this be confusing? Or would there be any case for overlap? (e.g., given that some plugin has trust levels, how to do those trust levels map to SourceCred? Or should they not?)

File: culture/trust_levels.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+# Motivation
+
+To achieve its goals when applied to real-world projects, SourceCred
+will need to be resistant to gaming by people who explicitly (but
+perhaps covertly) attempt to maximize their short-term or long-term
+profits at all other costs. This is because SourceCred is intended to be
+used to allocate scarce resources, and so actors will have incentives to
+acquire those resources and may have no natural incentive to contribute
+back to the community or behave in accordance with our preferred norms.
+
```
Comment (vsoch) [inline culture/trust_levels.md]: I would put a short list of examples here of what this could really mean, and this would be a helpful practice anyway to think about "How could SourceCred be gamed _without_ these trust levels?"

File: culture/trust_levels.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+However, full resistance to adversarial behavior will require
+substantial research, design, and implementation effort. Some of this
+effort cannot even begin until we have a fairly stable understanding of
+other parts of the pipeline. It doesn’t make sense to discuss defenses
+until we’ve defined the protocol in which everyone is to operate.
```
Comment (vsoch) [inline culture/trust_levels.md]: For the layperson reading, they probably don't know what the "pipeline" is referring to. And what constitutes a stable understanding of it? I think what you are simply saying is:

> However, full resistance to adversarial behavior will require substantial research, design, and implementation effort. Some of this effort cannot even begin until the SourceCred algorithm and associated tooling has gone through substantial testing.

or something like that?

File: culture/trust_levels.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+Furthermore, while the effort required to improve gaming resistance will
+surely bring us benefits in the long term, those benefits only
```
Comment (vsoch) [inline culture/trust_levels.md]: Yeah I'd definitely like to better understand what "gaming resistance" means by the time I get here!

File: culture/trust_levels.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+materialize when they actually prevent gaming, which can in turn only
+occur if people are trying to game the system in ways that our existing
+systems cannot sufficiently mitigate. Thus, as long as our community
+remains small enough that manual chastising and blacklisting can rein in
+the renegades, we’ll receive no short-term reward for any effort spent
+on this. We have *plenty* of other initiatives into which we’d prefer to
+invest our energy.
```
Comment (vsoch) [inline culture/trust_levels.md]: There should probably be some policy to be able to retroactively go back and "undo" some gaming that was discovered after the fact. I don't know how that would work, but it seems like a card that should be pullable.

File: culture/trust_levels.md [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+Therefore, we propose that we change our approach to trust over time,
+and lay out a plan to do so.
+
+# Trust levels defined
+
+We propose three trust levels, which characterize not the technical
+systems built by the SourceCred contributors but rather the composition
+of a community:
+
+  - **Trust level 3** (TL3): Collaborative. It is taken as [common
```
Comment (vsoch) [inline culture/trust_levels.md]: This is really really similar to Discourse, and I think even with the note above, people who are familiar might get confused. Is there some small tweak to the naming / something else that could differentiate the two?

File: culture/trust_levels.md [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+    knowledge][common-knowledge] that everyone in the community is
+    acting in good faith. People feel comfortable reflecting on their
```
Comment (vsoch) [inline culture/trust_levels.md]: What does this link to (common-knowledge)

File: culture/trust_levels.md [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+    own incentives: “am I subconsciously skewing my ‘votes’ to benefit
```
Comment (vsoch) [inline culture/trust_levels.md]: Should "Am" be capitalized?

Comment (vsoch) [reply culture/trust_levels.md]: Same with "How" below. What is trying to be said when you say "precious data?" Maybe valuable would work better here - precious hints on an obsessive or romantic level.

File: culture/trust_levels.md [hunk 11]
```diff
@@ -0,0 +1,108 @@
...
+  - **Trust level 1** (TL1): Alert. This community controls a scarce
+    resource. That resource is considered intrinsically valuable by some
+    people outside the community, who are willing to devote their time
+    to acquiring that resource without much thought for scruples.
+    Primarily distinguished from TL2 because gaming may now be the main
+    goal rather than an opportunistic “side hustle” on legitimate
+    contributions—noting, of course, that legitimate components may yet
+    be a key component of effective gaming strategies! Expect voting
+    cliques, [sybils], chains of obfuscated self-dealing, attempts to
+    corrupt structures of leadership and governance.
```
Comment (vsoch) [inline culture/trust_levels.md]: Link here for "sybils" not defined. I think I'm having a hard time understanding how the trust levels map to tangible permissions / actions. From reading, I'm trying to figure out via "Wary" and "Alert" if Level 1 is the most trusted or if it's 3. It doesn't seem super obvious / is a little confusing so we could probably improve on this, either with better description, name, or even colors.

File: culture/trust_levels.md [hunk 12]
```diff
@@ -0,0 +1,108 @@
...
+
+[common-knowledge]: https://en.wikipedia.org/wiki/Common_knowledge_(logic)
+[sybils]: https://en.wikipedia.org/wiki/Sybil_attack
+
```
Comment (vsoch) [inline culture/trust_levels.md]: oops, please disregard my comments about the missing links! I should have looked down here first :(

Comment (vsoch) [reply culture/trust_levels.md]: I meant that to be a smiley face. Doh. :)

File: culture/trust_levels.md [hunk 13]
```diff
@@ -0,0 +1,108 @@
...
+(As a point of syntax, note that the numbers decrease as the community
+is understood to be less trustworthy. That is, the number indicates the
+degree of trust, not the passage of time. This is for simplicity that
```
Comment (vsoch) [inline culture/trust_levels.md]: okay - so an entire community gets a trust level (not a person in it) and 1 is the least trusted?

File: culture/trust_levels.md [hunk 14]
```diff
@@ -0,0 +1,108 @@
...
+“high trust level should mean more trust”, as well as consistency with
+systems like DEFCON or Discourse’s own trust levels.)
+
+# Usage
+
+It’s important to recognize that trust level varies both with time and
+with community. As of late 2019, the SourceCred community is in TL3, but
+(e.g.) some cryptocurrency communities are surely in TL1.
+
```
Comment (vsoch) [inline culture/trust_levels.md]: I'd add parens here with context, like "community is in TL3 (high trust)... others in TL1 (low trust)." or something along those lines to still be teaching the reader about what the levels means.

File: culture/trust_levels.md [hunk 15]
```diff
@@ -0,0 +1,108 @@
...
+A given iteration of the SourceCred system will be appropriate for some
+subset of trust levels. As of late 2019, SourceCred is probably only
+appropriate for communities in TL3. As we begin to service communities
+at lower trust levels—perhaps including our own at a later time—we’ll
+need to anneal down, investing into systems that can handle the
+corresponding heat.
```
Comment (vsoch) [inline culture/trust_levels.md]: Any reason to not update to 2020, since it's being written here now?

File: culture/trust_levels.md (hunk 6 above)
Comment (s-ben) [reply culture/trust_levels.md]: I was able to infer the meaning of gaming resistance, and I think the average reader will too. But no harm in fleshing out.

File: culture/trust_levels.md (hunk 7 above)
Comment (s-ben) [reply culture/trust_levels.md]: One can always go back and change the scores (e.g. by deleting content), assuming the community is not fully decentralized, however Grain already distributed (e.g. as an ERC-20 token) cannot (and shouldn't be) be clawed back.

File: culture/trust_levels.md (hunk 11 above)
Comment (s-ben) [reply culture/trust_levels.md]: I think "sybils" and mapping to permissions/actions will be familiar to crypto peeps. But probably not to most OSS projects outside of that.

File: culture/trust_levels.md (hunk 15 above)
Comment (s-ben) [reply culture/trust_levels.md]: Per @decentralion's [nit for approval](https://github.com/sourcecred/docs/pull/24#pullrequestreview-366767991), 
suggest rewording this para to:

>It’s important to recognize that trust level varies both with time and with community. As of early 2020, SourceCred has only been actively tested with communities in TL3, is probably not appropriate for TL1, and should be used in TL2 with caution and close attention.

**Comment by decentralion** [comment]
I've decided to take over this PR. I re-wrote the guide a little bit to change the tone away from "sociotechnical specification" and towards the rest of the content on our docs. Core idea of TL's stays the same, though I added some examples.

I'd like to bias towards merging this rather than bikeshedding small wording discussions--let me know if you see serious issues, feel free to propose specific changes (i.e. with the wording you would use), but let's also try to get this merged. :)

**Comment by Beanow** [comment]
Given the current "fleshing out" mode of the docs repo, taking feedback through follow-up PRs / issues is OK.

For future ease of review, would have liked to see the commits in a way that makes it easy to see what your changes from Williams version looks like as a diff. In this case, by moving the file as a separate commit could do that. For reference, tried to manually diff here:

```diff
$ diff -u before.md after.md 
--- before.md	2020-05-08 15:48:22.420674002 +0200
+++ after.md	2020-05-08 15:47:58.457283496 +0200
@@ -1,70 +1,47 @@
-This post proposes a system of **trust levels** for SourceCred the
-system, which describe how we intend to handle the very real questions
-of adversarial actors and gaming resistance over different time scales.
-
-The ideas proposed herein have been discussed informally within our
-community, but with somewhat confusingly varying semantics. It may help
-to have this shared reference to point to.
-
-**Note:** The term *trust levels* as used in this post is largely
-separate from [Discourse trust levels][discourse-trust], which are a
-construct of our forum software and not related to the SourceCred
-protocol in any way.
-
-[discourse-trust]: https://blog.discourse.org/2018/06/understanding-discourse-trust-levels/
-
-# Motivation
-
-To achieve its goals when applied to real-world projects, SourceCred
-will need to be resistant to gaming by people who explicitly (but
-perhaps covertly) attempt to maximize their short-term or long-term
-profits at all other costs. This is because SourceCred is intended to be
-used to allocate scarce resources, and so actors will have incentives to
-acquire those resources and may have no natural incentive to contribute
-back to the community or behave in accordance with our preferred norms.
-
-However, full resistance to adversarial behavior will require
-substantial research, design, and implementation effort. Some of this
-effort cannot even begin until we have a fairly stable understanding of
-other parts of the pipeline. It doesn’t make sense to discuss defenses
-until we’ve defined the protocol in which everyone is to operate.
-
-Furthermore, while the effort required to improve gaming resistance will
-surely bring us benefits in the long term, those benefits only
-materialize when they actually prevent gaming, which can in turn only
-occur if people are trying to game the system in ways that our existing
-systems cannot sufficiently mitigate. Thus, as long as our community
-remains small enough that manual chastising and blacklisting can rein in
-the renegades, we’ll receive no short-term reward for any effort spent
-on this. We have *plenty* of other initiatives into which we’d prefer to
-invest our energy.
-
-Therefore, we propose that we change our approach to trust over time,
-and lay out a plan to do so.
-
-# Trust levels defined
-
-We propose three trust levels, which characterize not the technical
-systems built by the SourceCred contributors but rather the composition
-of a community:
-
-  - **Trust level 3** (TL3): Collaborative. It is taken as [common
-    knowledge][common-knowledge] that everyone in the community is
-    acting in good faith. People feel comfortable reflecting on their
-    own incentives: “am I subconsciously skewing my ‘votes’ to benefit
-    myself?”; “how have my motivations for votes, etc. changed over
-    time?”. When asking the same questions of others, we trust that the
-    responses are thoughtfully and truthfully given. Discussions of
-    accidental or subconscious gaming are treated as precious data to be
-    shared and analyzed, never as evidence of any kind of wrongdoing.
-
-  - **Trust level 2** (TL2): Wary. While the community’s core backbone
-    of contributors are still dedicated to the original ideals, perhaps
-    some new members are more interested in expanding their share of the
-    pie rather than the pie as a whole. Primarily distinguished from TL3
-    because good faith is no longer a uniform assumption. Expect skewed
-    voting records, underrecognition of others’ help, quarrels over
-    attribution or allocation.
+---
+title: 🎚 Trust Levels
+description: What level of "community trust" does SourceCred depend on
+---
+
+SourceCred is intended to be used by communities to divy up rewards, in the
+form of cred and grain. As such, SourceCred needs to be resilient to gaming by
+people who explicitly (but perhaps covertly) attempt to maximize their rewards
+at all other costs.
+
+The kind of gaming that will happen depends on the ambient level of trust and
+social cohesion within the community. A small community of like-minded and
+highly aligned people might see very little gaming, with most people explicitly
+working together to make sure everyone is fairly credited. On the other hand, a
+larger and more anonymous community may have many selfish actors who just want
+to increase their scores without regard for the broader mission.
+
+We've developed the conept of "Trust Levels" to try to categorize communities
+along this dimension of trust, so that we can reason about which communities
+SourceCred is ready to support, and what kinds of problems communities might
+encounter along the way.
+
+  - **Trust level 3** (TL3): Collaborative. This is the highest trust level.
+    It is taken as [common knowledge][common-knowledge] that everyone in the
+    community is acting in good faith. People feel comfortable reflecting on
+    their own incentives: “am I subconsciously skewing the cred to benefit
+    myself?”; “how have my motivations changed over time?”.
+    When asking the same questions of others, we trust that the responses are
+    thoughtfully and truthfully given. Discussions of accidental or
+    subconscious gaming are treated as precious data to be shared and analyzed,
+    never as evidence of any kind of wrongdoing.
+
+    *As an example, imaging group of friends launching a startup together, with
+    a bold vision of improving the world.*
+
+  - **Trust level 2** (TL2): Wary. The community has a core backbone of
+    contributors that are focused on common goals, but some people may be more
+    focused on optimizing their own rewards. Primarily distinguished from TL3
+    because good faith is no longer a uniform assumption. Expect skewed voting
+    records, underrecognition of others’ help, quarrels over attribution or
+    allocation.
+
+    *As an example, imagine the startup is now a growing company. The "core group"
+    is motivated by the vision, but some of the new hires just see it as a job.*
 
   - **Trust level 1** (TL1): Alert. This community controls a scarce
     resource. That resource is considered intrinsically valuable by some
@@ -77,29 +54,32 @@
     cliques, [sybils], chains of obfuscated self-dealing, attempts to
     corrupt structures of leadership and governance.
 
+    *As an example, imagine that the startup is now a large company. It has hired
+    a lot of ex-management-consultants and business school types.*
+
 [common-knowledge]: https://en.wikipedia.org/wiki/Common_knowledge_(logic)
 [sybils]: https://en.wikipedia.org/wiki/Sybil_attack
 
-(As a point of syntax, note that the numbers decrease as the community
-is understood to be less trustworthy. That is, the number indicates the
-degree of trust, not the passage of time. This is for simplicity that
-“high trust level should mean more trust”, as well as consistency with
-systems like DEFCON or Discourse’s own trust levels.)
 
-# Usage
+## Caveats
+
+This is only one dimension among several that will influence a community's
+experience with SourceCred.
+
+Using SourceCred healthily also requires emotional maturity,
+conflict resolution skills, and community leadership.
+
+## Usage
 
 It’s important to recognize that trust level varies both with time and
-with community. As of late 2019, the SourceCred community is in TL3, but
+with community. As of mid 2020, the SourceCred community is in TL3, but
 (e.g.) some cryptocurrency communities are surely in TL1.
 
-A given iteration of the SourceCred system will be appropriate for some
-subset of trust levels. As of late 2019, SourceCred is probably only
-appropriate for communities in TL3. As we begin to service communities
-at lower trust levels—perhaps including our own at a later time—we’ll
-need to anneal down, investing into systems that can handle the
-corresponding heat.
+A given iteration of the SourceCred system will be appropriate for some subset
+of trust levels. As of SourceCred 0.5.0, SourceCred has been tested in TL3, is
+experimentally available for Tl2, and should not be used in TL1.
 
-# References
+## References
 
   - This document was drafted on our Discourse:
     <https://discourse.sourcecred.io/t/trust-levels/490>
```

File: docs/concepts/trust_levels.md [hunk 16]
```diff
@@ -0,0 +1,88 @@
...
+at all other costs.
+
+The kind of gaming that will happen depends on the ambient level of trust and
+social cohesion within the community. A small community of like-minded and
+highly aligned people might see very little gaming, with most people explicitly
+working together to make sure everyone is fairly credited. On the other hand, a
+larger and more anonymous community may have many selfish actors who just want
+to increase their scores without regard for the broader mission.
+
+We've developed the conept of "Trust Levels" to try to categorize communities
```
Comment (Bex2594) [inline docs/concepts/trust_levels.md:18]: concept*

Comment (Bex2594) [inline docs/concepts/trust_levels.md:18]: Who is we? If we want to make the SC team more _personable_, we could say something like, "the SourceCred team has developed..." If we want SC to seem more _powerful_ in the way of seeming large, we could say, "SourceCred has developed..."

File: docs/concepts/trust_levels.md [hunk 17]
```diff
@@ -0,0 +1,88 @@
...
+along this dimension of trust, so that we can reason about which communities
```
Comment (Bex2594) [inline docs/concepts/trust_levels.md:19]: so we can determine

Comment (Bex2594) [inline docs/concepts/trust_levels.md:19]: what dimension?

File: docs/concepts/trust_levels.md [hunk 18]
```diff
@@ -0,0 +1,88 @@
...
+SourceCred is ready to support, and what kinds of problems communities might
+encounter along the way.
+
+  - **Trust level 3** (TL3): Collaborative. This is the highest trust level.
+    It is taken as [common knowledge][common-knowledge] that everyone in the
+    community is acting in good faith. People feel comfortable reflecting on
+    their own incentives: “am I subconsciously skewing the cred to benefit
```
Comment (Bex2594) [inline docs/concepts/trust_levels.md:26]: Am*
How*

Comment (Bex2594) [inline docs/concepts/trust_levels.md:26]: Am* 
How*

File: docs/concepts/trust_levels.md (hunk 16 above)
Comment (Bex2594) [inline docs/concepts/trust_levels.md:18]: "in an attempt to categorize..." (to try to doesn't make me want to trust the system very much and sounds less professional)

File: docs/concepts/trust_levels.md [hunk 19]
```diff
@@ -0,0 +1,88 @@
...
+    myself?”; “how have my motivations changed over time?”.
+    When asking the same questions of others, we trust that the responses are
+    thoughtfully and truthfully given. Discussions of accidental or
+    subconscious gaming are treated as precious data to be shared and analyzed,
+    never as evidence of any kind of wrongdoing.
+
+    *As an example, imaging group of friends launching a startup together, with
```
Comment (Bex2594) [inline docs/concepts/trust_levels.md:33]: imagine a*

File: docs/concepts/trust_levels.md [hunk 20]
```diff
@@ -0,0 +1,88 @@
...
+experience with SourceCred.
+
+Using SourceCred healthily also requires emotional maturity,
+conflict resolution skills, and community leadership.
+
+## Usage
+
+It’s important to recognize that trust level varies both with time and
+with community. As of mid 2020, the SourceCred community is in TL3, but
+(e.g.) some cryptocurrency communities are surely in TL1.
```
Comment (Bex2594) [inline docs/concepts/trust_levels.md:76]: Not sure I understand the usage of e.g. and referencing cryptocurrency communities in particular. Couldn't we just use a blanket statement?

File: docs/concepts/trust_levels.md [hunk 21]
```diff
@@ -0,0 +1,88 @@
...
+    thoughtfully and truthfully given. Discussions of accidental or
+    subconscious gaming are treated as precious data to be shared and analyzed,
+    never as evidence of any kind of wrongdoing.
+
+    *As an example, imaging group of friends launching a startup together, with
+    a bold vision of improving the world.*
+
+  - **Trust level 2** (TL2): Wary. The community has a core backbone of
+    contributors that are focused on common goals, but some people may be more
+    focused on optimizing their own rewards. Primarily distinguished from TL3
```
Comment (Bex2594) [inline docs/concepts/trust_levels.md:38]: some members* (this is purely my personal taste)

Comment (Bex2594) [inline docs/concepts/trust_levels.md:38]: Not sure I understand the sentence "primarily distinguished..."
Also, it is a sentence fragment.

File: docs/concepts/trust_levels.md [hunk 22]
```diff
@@ -0,0 +1,88 @@
...
+    because good faith is no longer a uniform assumption. Expect skewed voting
+    records, underrecognition of others’ help, quarrels over attribution or
```
Comment (Bex2594) [inline docs/concepts/trust_levels.md:40]: and quarrels

File: docs/concepts/trust_levels.md [hunk 23]
```diff
@@ -0,0 +1,88 @@
...
+    allocation.
+
+    *As an example, imagine the startup is now a growing company. The "core group"
+    is motivated by the vision, but some of the new hires just see it as a job.*
+
+  - **Trust level 1** (TL1): Alert. This community controls a scarce
+    resource. That resource is considered intrinsically valuable by some
+    people outside the community, who are willing to devote their time
+    to acquiring that resource without much thought for scruples.
+    Primarily distinguished from TL2 because gaming may now be the main
```
Comment (Bex2594) [inline docs/concepts/trust_levels.md:50]: This is a sentence fragment and a run-on, clearer sentences would definitely help here

File: docs/concepts/trust_levels.md [hunk 24]
```diff
@@ -0,0 +1,88 @@
...
+    goal rather than an opportunistic “side hustle” on legitimate
+    contributions—noting, of course, that legitimate components may yet
+    be a key component of effective gaming strategies! Expect voting
+    cliques, [sybils], chains of obfuscated self-dealing, attempts to
```
Comment (Bex2594) [inline docs/concepts/trust_levels.md:54]: and attempts

Comment (Bex2594) [inline docs/concepts/trust_levels.md:54]: "chains of obfuscated self-dealing" could maybe be explained in more accessible language
